code/game/objects/items/his_grace.dm

```diff
@@ -0,0 +1,262 @@
+	else
+		. += "<span class='his_grace'>[src] is latched closed.</span>"
+
+/obj/item/his_grace/relaymove(mob/living/user, direction) //Allows changelings, etc. to climb out of Him after they revive, provided He isn't active
+	if(!awakened)
+		user.forceMove(get_turf(src))
+		user.visible_message("<span class='warning'>[user] scrambles out of [src]!</span>", "<span class='notice'>You climb out of [src]!</span>")
+
+/obj/item/his_grace/process()
+	if(!bloodthirst)
+		drowse()
+		return
+	if(bloodthirst < HIS_GRACE_CONSUME_OWNER && !ascended)
+		adjust_bloodthirst(1 + FLOOR(length(contents) * 0.5, 1)) //Maybe adjust this?
+	else
+		adjust_bloodthirst(1) //don't cool off rapidly once we're at the point where His Grace consumes all.
+	var/mob/living/master = get_atom_on_turf(src, /mob/living)
+	var/list/held_items = list()
+	if(istype(master))
+		held_items += master.l_hand
+		held_items += master.r_hand
+	if(istype(master) && (src in held_items))
+		switch(bloodthirst)
+			if(HIS_GRACE_CONSUME_OWNER to HIS_GRACE_FALL_ASLEEP)
+				master.visible_message("<span class='boldwarning'>[src] turns on [master]!</span>", "<span class='his_grace big bold'>[src] turns on you!</span>")
+				do_attack_animation(master, null, src)
+				master.emote("scream")
+				master.remove_status_effect(STATUS_EFFECT_HISGRACE)
+				flags &= ~NODROP
+				master.Weaken(3)
+				master.adjustBruteLoss(1000)
+				playsound(master, 'sound/effects/splat.ogg', 100, FALSE)
+			else
+				master.apply_status_effect(STATUS_EFFECT_HISGRACE)
+		return
+	forceMove(get_turf(src)) //no you can't put His Grace in a locker you just have to deal with Him
+	if(bloodthirst < HIS_GRACE_CONSUME_OWNER)
+		return
+	if(bloodthirst >= HIS_GRACE_FALL_ASLEEP)
+		drowse()
+		return
+	var/list/targets = list()
+	for(var/mob/living/L in oview(2, src))
+		targets += L
+	if(!length(targets))
+		return
+	var/mob/living/L = pick(targets)
+	step_to(src, L)
+	if(Adjacent(L))
+		if(!L.stat)
+			L.visible_message("<span class='warning'>[src] lunges at [L]!</span>", "<span class='his_grace big bold'>[src] lunges at you!</span>")
+			do_attack_animation(L, null, src)
+			playsound(L, 'sound/weapons/smash.ogg', 50, TRUE)
+			playsound(L, 'sound/misc/desceration-01.ogg', 50, TRUE)
+			L.adjustBruteLoss(force)
+			adjust_bloodthirst(-5) //Don't stop attacking they're right there!
+		else
+			consume(L)
+
+/obj/item/his_grace/proc/awaken(mob/user) //Good morning, Mr. Grace.
+	if(awakened)
+		return
+	awakened = TRUE
+	user.visible_message("<span class='boldwarning'>[src] begins to rattle. He thirsts.</span>", "<span class='his_grace'>You flick [src]'s latch up. You hope this is a good idea.</span>")
+	name = "His Grace"
+	desc = "A bloodthirsty artifact created by a profane rite."
+	gender = MALE
+	adjust_bloodthirst(1)
+	force_bonus = HIS_GRACE_FORCE_BONUS * length(contents)
+	playsound(user, 'sound/effects/pope_entry.ogg', 100)
```

Comment:
Doesn't actually make a difference sound wise, but I think this should be played from His Grace instead of His wielder.